### ISSUE TITLE
Update to USWDS 2.13.2

### ISSUE THREAD

**Issue by mejiaj** [GSA/accessibility-for-teams#98]
The currently installed version is 2.8.1, which was released on Aug 17, 2020. The version should be updated to improve security, accessibility, performance, and other enhancements.

Some notable improvements since 2.8.1

- Identifier component introduced
- Initializer script to prevent flash of unstyled content (noticeable in banner on load)
- Improved icons
- Table display improvements
- Improved accessible color setting
- 404 page template
- Modals
- Header a11y improvements
- SASS deprecation warnings gone

[See full comparison from 2.13.2 and 2.8.1 [github.com]](https://github.com/uswds/uswds/compare/v2.8.1...v2.13.2)

**Comment by nick-mon1** [comment]
Resolved by #100
